api/v2/serializers/details/user.py

```diff
@@ -3,19 +3,18 @@
 from api.v2.serializers.fields.base import UUIDHyperlinkedIdentityField
 
 
-class UserSerializer(serializers.HyperlinkedModelSerializer):
-    url = UUIDHyperlinkedIdentityField(
-        view_name='api:v2:atmosphereuser-detail',
-    )
-
+class UserSerializer(serializers.ModelSerializer):
+    """
+    This serializer used for 'user lookups'
+    """
     class Meta:
         model = AtmosphereUser
         fields = (
             'id',
             'uuid',
-            'url',
+            # 'url',
```

Comment:
Are these now automatically added?

Reply:
Or are they no longer necessary?

Reply:
They are not automatically added. This serializer will be accessible by end-users and the only information required is `id` and `username`. Additional fields, like url, are not being returned to avoid excess overhead..